server/models/Checkout.js

```diff
@@ -35,7 +35,23 @@ const checkoutDBSchema = mongoose.Schema({
 
 const CheckoutDB = mongoose.model('CheckoutDB', checkoutDBSchema);
 
-if (!CheckoutDB) {
+let checkIfDBEmpty = () => {
```

Comment:
i think you might be overcomplicating your approach here. if i understand your current challenge and how this code works, you reseed your database everytime you start your server. does it make sense to separate seeding code and helper functions?

Reply:
Good point I'll tinker with that now you're right they're bundled in one huge setup